data/templates/default.latex

```diff
@@ -247,8 +247,8 @@ $if(beamer)$
 \def\fnum@table{\tablename~\thetable}
 \makeatother
 $else$
-% Fix footnotes in tables (requires footnote package)
-\IfFileExists{footnote.sty}{\usepackage{footnote}\makesavenoteenv{longtable}}{}
+% Allow footnotes in tables
+\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}\makesavenoteenv{longtable}
```

Comment:
Maybe put `\makesavenoteenv{longtable}` on a new line, since it is no longer part of the `IfFileExists`??

Reply:
Done; thanks!